rasa/cli/x.py

```diff
@@ -350,17 +349,15 @@ def rasa_x(args: argparse.Namespace) -> None:
 
     _configure_logging(args)
 
-    if version.parse(rasa.version.__version__) >= version.parse("3.0.0"):
```

Comment:
By virtue of this code being only in Rasa OSS 3.x, it's guaranteed that the version of OSS is above 3.0, so I removed the check.

Re. Rasa X 1.1 support for OSS 3.0: This does not influence the OSS support of the `rasa x` command, since it starts localmode, which remains unsupported.

Reply:
I see, I got confused by the fact that the last `with pytest.warns(None)` check with version `2.8.0` doesn't fail

Reply:
Yeah I'm not entirely sure why that's there - seems like that part of the test is pointless in a 3.x codebase